src/EnergyPlus/EPVector.hh

```diff
@@ -126,7 +140,9 @@ private:
 template <> struct EPVector<bool> : private std::vector<std::uint8_t>
```

Comment:
Boolean vectors require a little more work, which is not surprising, but it's basically the same as the above with a few additional changes to accommodate the `noexcept` labeling.